pkg/evs/controllerserver.go

```diff
@@ -266,18 +269,181 @@ func (cs *ControllerServer) ListVolumes(_ context.Context, req *csi.ListVolumesR
 
 func (cs *ControllerServer) CreateSnapshot(_ context.Context, req *csi.CreateSnapshotRequest) (
 	*csi.CreateSnapshotResponse, error) {
-	return nil, status.Error(codes.Unimplemented, "")
+	klog.Infof("CreateSnapshot called with request %v", *req)
```

Comment:
suggest to use `protosanitizer.StripSecrets(*req)`, the log is easier to read

Reply:
done